src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -399,6 +399,26 @@ public void FailedToLoadEnvironmentVariables(string ex, string appDomainName = "
             this.WriteEvent(32, ex, this.nameProvider.Name);
         }
 
+        // Verbosity is Error - so it is always sent to portal; Keyword is Diagnostics so throttling is not applied.
+        [Event(33,
+            Message = "Metrics Extractor should be used before Sampling Processors or any other Telemetry Processors that might filter out Telemetry Items. Otherwise, extracted metrics may be incorrect.",
+            Level = EventLevel.Error,
+            Keywords = Keywords.Diagnostics | Keywords.UserActionable)]
+        public void MetricExtractorAfterSampling_Error(string appDomainName = "Incorrect")
+        {
+            this.WriteEvent(33, this.nameProvider.Name);
+        }
+
+        // Verbosity is Verbose - so it is always sent to portal; Keyword is Diagnostics so throttling is not applied.
+        [Event(34,
+            Message = "Metrics Extractor should be used before Sampling Processors or any other Telemetry Processors that might filter out Telemetry Items. Otherwise, extracted metrics may be incorrect.",
```

Comment:
> MetricExctractor will ignore a telemetry item of type {0} with the SamplingPercentage {1}

This incorrectly describes the issue. Yes, we will ignore it, but it indicates a bigger problem that the current message describes. SO I will leave the current message.

> This will not be send to portal, only analyzed by support personnel

I have copied the comment from elsewhere so it must be wrong there too. Will remove the incorrect comment.